components/CommunityActivity/CommunityActivity.tsx

```diff
@@ -21,15 +22,22 @@ export function CommunityActivity({ account }: CommunityActivityProps) {
 
   return (
     <div className={styles.wrapper}>
-      <h2>✨ XP Earned</h2>
-      <ol>
-        {scoreChanges.map((event) => (
-          <li data-testid={`event-${event.id}`} key={event.id}>
-            {event.category.name} 1XP{' '}
-            {new Date(event.timestamp * 1000).toLocaleDateString()}
-          </li>
-        ))}
-      </ol>
+      <Fieldset legend="✨ XP Earned">
+        {scoreChanges.length === 0 && (
+          <span>No XP earned by this address yet.</span>
+        )}
+        {scoreChanges.length > 0 && (
+          <ol className={styles.list}>
+            {scoreChanges.map((event) => (
+              <li data-testid={`event-${event.id}`} key={event.id}>
+                {event.category.name} 1XP{' '}
```

Comment:
ok first thing i notice, and i forgot to mention this so its on me, is that the `CategoryScoreChange` entity has an oldScore and newScore field